kie-internal/src/main/java/org/kie/internal/utils/ChainedProperties.java

```diff
@@ -121,25 +114,14 @@ public ChainedProperties(String confFileName,
                         this.props, confClassLoader );
 
         confClassLoader = Thread.currentThread().getContextClassLoader();
-        if ( confClassLoader != null && confClassLoader != classLoader ) {
-            loadProperties( getResources( "META-INF/drools." + confFileName,
-                                          confClassLoader ),
-                            this.props, confClassLoader );
-            loadProperties( getResources( "/META-INF/drools." + confFileName,
-                                          confClassLoader ),
-                            this.props, confClassLoader );
-        }
-        
-        confClassLoader = ClassLoader.getSystemClassLoader();
-        if ( confClassLoader != null && confClassLoader != classLoader ) {
-            loadProperties( getResources( "META-INF/drools." + confFileName,
-                                          confClassLoader ),
-                            this.props, confClassLoader );
-            loadProperties( getResources( "/META-INF/drools." + confFileName,
-                                          confClassLoader ),
-                            this.props, confClassLoader );
+        loadPropertiesFromClassLoader( confFileName, classLoader, confClassLoader );
+
+        try {
+            confClassLoader = ClassLoader.getSystemClassLoader();
+            loadPropertiesFromClassLoader( confFileName, classLoader, confClassLoader );
```

Comment:
I wouldn't wrap the loadPropertiesFromClassLoader() in the try-catch as that try-catch will eat any exceptions that happen in there.